api/openapi.yaml

```diff
@@ -94,28 +93,30 @@ paths:
               $ref: ./model/Login.v1.yaml
       responses:
         '200':
-          description: Ok.
+          description: Ok. The response contains user authenticated data.
           content:
             application/json:
               schema:
                 $ref: ./model/User.v1.yaml
         '401':
           $ref: '#/components/responses/Unauthorized'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
         default:
           $ref: '#/components/responses/DefaultResponse'
-      summary: Create a login request for an user
-  '/users/{userId}/organizations/connected':
+      summary: Create a login request for an user.
+  /user/{userId}/connected:
     parameters:
       - $ref: '#/components/parameters/userId'
-    get:
+    get: #TODO
       tags:
         - stalker mobile app
         - stalker web app
       description: Get the list of organizations that a user has connected to.
-      operationId: getSpecificUserSubscriptions
+      operationId: getSpecificUserConnectedOrganizations
       responses:
         '200':
-          description: The list of the ids of the organizations the user is connected to.
+          description: Ok. The response contains the list of the ids of the organizations who the user is connected to.
```

Comment:
"who the user is connected to." suona male, e "who" si usa solo con le persone. Meglio "[...] the ids of the organizations which the user is connected to", o ancora meglio "[...] the ids of the organizations to which the user is connected".

Reply:
Comunque il minimo sindacale è mettere which (o that) invece di who.